### PR TITLE
update: team page

### PR DESCRIPTION
**Pull request by tusharnankani**
This PR includes:

- Update PD's alt tag.
- Add TN's website.
- Make socials consistent for SS.

### CODE REVIEW

**Comment by tusharnankani** [comment]
Nice, forgot to run Prettier 👌

File: team.html [hunk 1]
```diff
@@ -354,7 +354,7 @@ <h1 class="clickable">The OTC Team</h1>
 					>
 						<img
 							src="static/img/globe.svg"
-							alt="Harsh Kapadia's website: https://pranavdani.github.io"
+							alt="Pranav Dani's website: https://pranavdani.github.io"
```
Comment (tusharnankani) [inline team.html]: Harsh yaar

Comment (HarshKapadia2) [reply team.html]: Oops. My bad. I'll correct this.

Comment (HarshKapadia2) [reply team.html]: Oh you've already corrected it. Thank you. My bad.

**Comment by HarshKapadia2** [comment]
Well if these changes are being made, I might as well add a few more.

Additional changes being made:
- Undo @tusharnankani's 'Janitor' designation change in the previous commit to keep a flat organisation structure (as discussed umpteen number of times).
- Update @PranavDani's web site.
- Add pronouns for everyone.
- Add @KartikSoneji's GitHub handle.
- Update my picture.

> I should not have made so many changes in one commit, but they are all trivial housekeeping changes that don't need to be reviewed thoroughly and I was too lazy to split all changes into different commits. We are going to 'squash and merge' anyways.

**Comment by tusharnankani** [comment]
> Undo @tusharnankani's 'Janitor' designation change in the previous commit to keep a flat organisation structure (as discussed umpteen number of times).

I thought you will miss it :)

**Comment by tusharnankani** [comment]
Oh you made the changes, I was just getting started. 
Thank you, though!

**Comment by HarshKapadia2** [comment]
Please review this and merge it if everything seems okay.

**Comment by tusharnankani** [comment]
Although I would ask you to add a snapshot or potentially a link of updated website.
Helps save time from local pulls and setup.

**Comment by HarshKapadia2** [comment]
Makes sense. Thank you.

This is how it looks:

**Comment by tusharnankani** [comment]
Looks good. 
Let's merge!

**Pull request merged**